File: helpers/md5helper.pl
#!/usr/bin/env perl
use strict;
use warnings;
use Digest::MD5;

my $file = shift or die "Usage: $0 <file>\n";
open my $fh, '<', $file or die "File not found\n";
binmode($fh);
my $md5 = Digest::MD5->new->addfile($fh)->hexdigest;
close $fh;
print "$md5  $file\n";

File: helpers/md5helper.php
#!/usr/bin/env php
<?php
if ($argc != 2) {
    fwrite(STDERR, "Usage: {$argv[0]} <file>\n");
    exit(1);
}
$file = $argv[1];
if (!file_exists($file)) {
    fwrite(STDERR, "File not found: $file\n");
    exit(1);
}
$hash = md5_file($file);
echo $hash . "  " . $file . PHP_EOL;
?>
